custom_dc/troubleshooting.md

```diff
@@ -43,12 +43,17 @@ Failed to create metadata: failed to create secret manager client: google: could
 
 This indicates that you have not specified API keys in the environment file. Follow procedures in [One-time setup steps](/custom_dc/quickstart.html#setup) to obtain and configure API keys.
 
+{: #schema-check-failed}
 ### "SQL schema check failed"
 
-This error indicates that there is a problem with the database schema. Check for the following additional error:
+This error indicates that there has been an update to the database schema, and you need to update your data management job to be compatible with the new schema. To update the schema:
```

Comment:
Nit: Rather than "you need to update your data managemebt job", it's that "you need to update your database schema...by re-running the data management job".

Reply:
Done.